bitbybit-tests/tests/bitfield_tests.rs

```diff
@@ -637,20 +637,20 @@ fn bitfield_with_u8_enum() {
 
     assert_eq!(
         Ok(NonExhaustiveEnum::Zero),
-        BitfieldWithIndexedEnums::new_with_raw_value(0b00_00000000_000000).val8()
```

Comment:
These changes are great for the hex values elsewhwere in this file, but for binary the old style worked better as its separators lined up with the bitfield definition

Reply:
I noticed that, but thought it was a little gimmicky and doesn't add much readability. If you want it the old way I can turn it back though.

Reply:
Nevermind, this is fine!